server/testutils/containere2e/containere2e.go

```diff
@@ -14,7 +17,25 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var buildPluginOnce sync.Once
+
+func buildPlugin(t *testing.T) {
```

Comment:
Build the plugin especially for these tests, and only do it once.

Reply:
I like this.